pages.de/windows/cipher.md

```diff
@@ -0,0 +1,16 @@
+# cipher
+
+> Verschlüsselt oder entschlüsselt Dateien auf NTFS-Laufwerken.
+> Weitere Informationen: <https://learn.microsoft.com/de-de/windows-server/administration/windows-commands/cipher>.
+
+- Verschlüsseln einer Datei oder eines Verzeichnisses:
+
+`cipher /e:{{Pfad/zur/Datei_oder_Verzeichnis}}`
```

Comment:
All command arguments should be lowercase and we decided on `zu` instead of `zur` some time ago.
```suggestion
`cipher /e:{{pfad/zu/datei_oder_verzeichnis}}`
```